sys/mips/mips/trap.c

```diff
@@ -1030,10 +1030,9 @@ trap(struct trapframe *trapframe)
 #if defined(KDTRACE_HOOKS) || defined(DDB)
 	case T_BREAK:
 #ifdef KDTRACE_HOOKS
-		if (!usermode && dtrace_invop_jump_addr != 0) {
-			dtrace_invop_jump_addr(trapframe);
+		if (!usermode && dtrace_invop_jump_addr != NULL &&
+			dtrace_invop_jump_addr(trapframe) == 0)
```

Comment:
you should use two tabs and four spaces before dtrace_invop_jump_addr(..)

Reply:
are there rules for an autoformatter somewhere?

Reply:
The .clang-format config file kinda works but is not perfect